vnc/vnc.c

```diff
@@ -1422,6 +1447,11 @@ lib_mod_set_param(struct vnc *v, const char *name, char *value)
     {
         v->delay_ms = g_atoi(value);
     }
+    else if (g_strcasecmp(name, "guid") == 0)
```

Comment:
I don't understand. Should users configure it? If not, who should configure it? If nobody should configure it, why is it read from the config file?